From 8b86041f824c7e7aa6ddf2c9890d638536723798 Mon Sep 17 00:00:00 2001
From: rnaidu <ramya.v.naidu@intel.com>
Date: Wed, 5 Dec 2018 04:06:00 -0500
Subject: [PATCH 1/1] Abnormal behavior observed in local media player on
 changing the display size

On changing the display size, local media player closes and music being played will be closed

Change-Id: Ic7611b1cc9d03df8fd4fbd70240ffbc2bad3703c
Tracked-On: OAM-71743
Signed-off-by: rnaidu <ramya.v.naidu@intel.com>
---
 AndroidManifest.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 4330248..62b7e76 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -20,7 +20,7 @@
 
     <uses-sdk
         android:minSdkVersion="23"
-        android:targetSdkVersion="23" />
+        android:targetSdkVersion="24" />
 
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.INTERACT_ACROSS_USERS_FULL"/>
-- 
1.9.1

